app/views/shared/_cancel_action_modal.html.slim

```diff
@@ -1,4 +1,6 @@
 - app_flow = idv ? 'idv' : 'sign_up'
+- warning_qualifier = t("#{app_flow}.cancel.warning_qualifier")
+- warning_qualifier = "<span class='italic'>#{warning_qualifier}</span>"
```

Comment:
I was having issues escaping the second set of double quotes`"`. I can try again if you think it should be simplified.